web/src/routes/wohnen/components/__tests__/OfferDetail.spec.js

```diff
@@ -62,6 +62,7 @@ describe('OfferDetail', () => {
       <OfferDetail offer={offer} />
     )
 
+    // $FlowFixMe
```

Comment:
Add comments to the flowfixme in this file

Reply:
I don't want to put any effort in the wohnen offer tbh. :grin: